lib/Backend/Encoder.cpp

```diff
@@ -642,6 +642,12 @@ Encoder::ShortenBranchesAndLabelAlign(BYTE **codeStart, ptrdiff_t *codeSize)
     BYTE* buffEnd = buffStart + *codeSize;
     ptrdiff_t newCodeSize = *codeSize;
 
+    RelocList* relocList = m_encoderMD.GetRelocList();
+
+    if (relocList == nullptr)
+    {
+        return false;
```

Comment:
It might still be nice to keep an `Assert(false)` here so it is easier to find and debug, unless this is considered normal behavior now.